unicon_backend/migrations/versions/0dbee5c8d237_add_permission_and_groups.py

```diff
@@ -0,0 +1,79 @@
+"""add permission and groups
```

Comment:
(opinionated nit)

would it better if we squash all access-control migrations into a single file? i am thinking we could add the migration to add unique constraint to username to the base file (`4303244cdb7a_initialize_tables.py`)

since we are still not maintaining a production deployment with user data, we could properly structure our migrations so that we have a better idea of what each migration file is doing at **a feature level**. imo we can still afford to make fixes to older migration files by updating instead of creating new migration files to keep things clean

Reply:
good idea! easier to autogenerate too and can't have irreversible migrations if there are almost no migrations 🎉